src/components/button/button.jsx

```diff
@@ -11,6 +11,11 @@ const ButtonComponent = ({
     iconSrc,
     onClick,
     children,
+    arrowLeft,
+    arrowRight,
+    primary,
+    secondary,
```

Comment:
What happens, if you set `primary` and `secondary` at the same time?
Maybe think about switching variants via a `style` prop?
See: https://github.com/wdr-data/code4maus/blob/develop/src/components/modal/modal.jsx#L77